### ISSUE TITLE
Unable to de-drm Amazon ebook

### ISSUE THREAD

**Issue by phrooo** [noDRM/DeDRM_tools#51]
### Question / bug report

Hello
Pretty much what the title says, I've tried to de-drm a kindle ebook, but it doesn't seem to be working. 

### Which version of Calibre are you running?

4.17

### Which version of the DeDRM plugin are you running?

10.0.2

### If applicable, which version of the Kindle software are you running?

1.17.0

### Log output

```log
calibre 4.17  embedded-python: True is64bit: False
Windows-7-6.1.7601-SP1 Windows ('32bit', 'WindowsPE')
32bit process running on 64bit windows
('Windows', '7', '6.1.7601')
Python 2.7.16+
Windows: ('7', '6.1.7601', 'SP1', u'Multiprocessor Free')
Interface language: pt_BR
Successfully initialized third party plugins: Gather KFX-ZIP (from KFX Input) (1, 41, 0) && DeDRM (10, 0, 2) && Package KFX 

(from KFX Input) (1, 41, 0) && KFX metadata reader (from KFX Input) (1, 41, 0) && KFX Input (1, 41, 0)
calibre 4.17  embedded-python: True is64bit: False
Windows-7-6.1.7601-SP1 Windows ('32bit', 'WindowsPE')
32bit process running on 64bit windows
('Windows', '7', '6.1.7601')
Python 2.7.16+
Windows: ('7', '6.1.7601', 'SP1', u'Multiprocessor Free')
Interface language: pt_BR
Successfully initialized third party plugins: Gather KFX-ZIP (from KFX Input) (1, 41, 0) && DeDRM (10, 0, 2) && Package KFX 

(from KFX Input) (1, 41, 0) && KFX metadata reader (from KFX Input) (1, 41, 0) && KFX Input (1, 41, 0)
Turning on automatic hidpi scaling
devicePixelRatio: 1.0
logicalDpi: 96.0 x 96.0
physicalDpi: 72.0567375887 x 71.8867924528
Using calibre Qt style: True
[0.00] Starting up...
[0.00] Showing splash screen...
[0.39] splash screen shown
[0.39] Initializing db...
[0.41] db initialized
[0.41] Constructing main UI...
[7.79] main UI initialized...
[7.79] Hiding splash screen
[102.99] splash screen hidden
[102.99] Started up in 102.99 seconds with 435 books
stdout+stderr from file dialog helper: ['', '']
piped data from file dialog helper: ['\xb1\xb3\xea5\xb6wO7\xf3\xfa\xc6\xbaD

\xa8\xd8\x86\x12\x06\xb4\xb1\xb7\xf5\xb4\x85\xc3\x93\xea\x8f\xa3\x91\x04\xb0', 'C:\\Users\\Win 3\\Desktop\

\B092335C53_EBOK.azw']
Gather KFX-ZIP (from KFX Input) 1.41.0: Importing C:\Users\Win 3\Desktop\B092335C53_EBOK.azw
Gather KFX-ZIP (from KFX Input): File is not KFX format
DeDRM v10.0.2: Trying to decrypt B092335C53_EBOK.azw
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "site-packages\calibre\customize\ui.py", line 172, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 888, in run
  File "calibre_plugins.dedrm.__init__", line 782, in KindleMobiDecrypt
  File "site-packages\calibre\customize\zipplugin.py", line 185, in load_module
  File "calibre_plugins.dedrm.k4mobidedrm", line 72, in <module>
ImportError: No module named html.entities
C:\Program Files (x86)\Calibre2\\app\pylib.zip\dateutil\parser\_parser.py:1177: UnicodeWarning: Unicode equal comparison 

failed to convert both arguments to Unicode - interpreting them as being unequal
```

**Comment by phrooo** [comment]
> For that version of calibre, try DeDRM 6.8.1: https://github.com/apprenticeharper/DeDRM_tools/releases/tag/v6.8.1

Thank you, this worked perfectly!

**Comment by noDRM** [comment]
Completely untested, but this bug should now also be fixed with e4fe032e4719e87e004b2f36e433051bf3345b85, so the next release should have this working on Calibre 4 again, too.
